app/src/main/java/com/example/demo/kotlin_practice/classesExamples/DemoAbstract.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.demo.kotlin_practice.classesExamples
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#naming-rules

Just concatenate the words in package name, do not use _ or camel case

Reply:
Rename `kotlin_practice` and `classesExamples`